package.json

```diff
@@ -36,6 +37,7 @@
     "jest": "^20.0.4",
     "prettier": "~1.6.0",
     "rollup": "^0.49.1",
+    "typescript": "^2.5.2",
```

Comment:
@Swizz Is the common practice to include the entire typescript module?

Reply:
This is because we are using `tsc` the typescript compiler to validate our DTS